docs/02-analyze/02-learning-goals.adoc

```diff
@@ -64,7 +64,7 @@ Typische Praktiken der Ist-Analyse kennen und für die jeweilige Situation, das
 [[LG-2-1]]
 ==== LG 2-1: Basics of the analysis to distinguish “problem” from “solution”
 
-* Distinguish “problem analysis” from “problem solution”
+* Distinguish between “analysis of problems” and “solving problems”
```

Comment:
Wenn, dann solltest du das gleich schreiben: 
`"analyzing problems" and "solving problems"`